tests/test_loader.py

```diff
@@ -277,6 +277,41 @@ async def test_async_get_platform_caches_failures_when_component_loaded(
         assert await integration.async_get_platform("light") == hue_light
 
 
+async def test_async_get_platforms_caches_failures_when_component_loaded(
```

Comment:
Should add a test like I did for async_get_comp to see if it uses executor or loop